qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data
+  file = File.new('data.txt', 'r:UTF-8')
+  file.read
+end
+
+def weather_data(town, data)
+  monthly_rainfall = data.split("\n").find { |data_row| data_row.match "#{town}:" }
+  return monthly_rainfall.scan(/\d*\.\d/).map(&:to_f) unless monthly_rainfall.nil?
```

Comment:
`if` + `present?` looks more readable here
and feel free to get rid of the `return`